### DISCUSSION TITLE
Student Developer Pack - I had an approval, but I can't use it. Can you help me?

### DISCUSSION THREAD

**Discussion by mkkkpln** [community/community#70018] [GitHub Education]
### Hi everyone,

How can I receive all of the benefits of Student Developer Pack? I approved on the 11th of September. When I will have it?

**Comment by IndraTensei** [comment]
Hi there,
Did you receive a mail from them? 
Something like this 
![image](https://github.com/community/community/assets/69427966/d09109ac-9647-4b8e-9b84-913fff0f95ad)

Usually this comes within 3 days or within 2 weeks, don't really know about your case.
You can make a ticket in https://support.github.com/tickets/ and inquire about it.

**Reply by mkkkpln** [reply]
![image](https://github.com/community/community/assets/72685907/83d81a35-5ea3-49a4-af1d-2baeb7f04c18)
I have this one

**Reply by IndraTensei** [reply]
Open a ticket here and inquire about it, it will get solved
https://support.github.com/tickets/
